tests/framework/extensions/provisioning/verify.go

```diff
@@ -43,7 +43,7 @@ const (
 	logMessageKubernetesVersion = "Validating the current version is the upgraded one"
 	hostnameLimit               = 63
 	machineNameAnnotation       = "cluster.x-k8s.io/machine"
-	publicIPAnnotation          = "rke2.io/external-ip"
+	publicRKE2IPAnnotation      = "rke2.io/external-ip"
```

Comment:
Do we even use this anymore?

Reply:
I use it in line 443 `PublicIPAddress: node.Annotations[publicRKE2IPAnnotation],`